codegen/src/wasm_interface.rs

```diff
@@ -44,7 +44,7 @@ type ConfigUsed = PolkadotConfig;
 
 #[wasm_bindgen]
 pub struct Client {
-    kind: ClientKind,
+    _kind: ClientKind,
```

Comment:
Could we remove the `_kind` field entirely? Do we plan to use it again shortly in the future?